File: exercicios em C/q30.cpp
#include <stdio.h>

int main(void){
	int soma, preco, i, n;
	printf ("\n\nEste programa determina o preco total de n obras de um museu.");
	printf ("\n\nDigite a quantidade de obras a qual voce quer calcular o preco: ");
	scanf ("%d", &n);
	
	for(i = 1; i <= n; i++){
		printf("\n\nDigite o preco da %d obra: ", i);
		scanf("%d", &preco);
		soma = soma + preco;
	}
	
	printf ("\n\nO valor total das obras e: %d", soma);
	printf ("\n\nObrigado por utilizar nosso programa!");
	return 0;
}

File: exercicios em Python/q30.py
import os

print ("\n\nEste programa determina o preco total de n obras de um museu.")
n = int(input("\nDigite a quantidade de obras a qual voce quer calcular o preco: "))
i = 1
soma = 0
while i <= n:
        print("\nDigite o preco da ",i," obra: ")
        preco = int(input(""))
        soma = soma + preco
        i = i + 1

print("\n\nO valor total das obras e: ",soma)
print("\n\nObrigado por utilizar nosso programa!")
os.system("pause")
